lgc/include/lgc/state/PipelineState.h

```diff
@@ -132,6 +132,10 @@ class PipelineState final : public Pipeline {
   // Set shader stage mask
   void setShaderStageMask(unsigned mask) override final { m_stageMask = mask; }
 
+  // Set whether pre-rasterization part has a geometry shader
+  // NOTE: Only applicable in the part pipeline compilation mode.
+  void setPreRastHasGs(bool preRastHasGs) override final { m_preRastHasGs = preRastHasGs; }
```

Comment:
Prefer to use Raster as the abbreviation
setPreRastHasGs -> setPreRasterHasGs
getPreRastHasGs -> getPreRasterHasGs